src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Credential/ICredentialProvider.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using System;
+using System.Threading;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Credential
+{
+    public interface ICredentialProvider
```

Comment:
What's the difference between this and the service?

Reply:
Removed service and updated this to be a generic.  Credentials are still being worked on and can be ignored for now.